### ISSUE TITLE
Improvements and other ideas for training detection

### ISSUE THREAD

**Issue by nikk-nikaznan** [SainsburyWellcomeCentre/crabs-exploration#96]
- [ ] [imgaug](https://github.com/aleju/imgaug/issues/406#issuecomment-733468745) -- #141 
- [x] loss function and revise the loss plot in tensorboard
- [x] learning rate schedule -- optuna #136 - implemented but pause to prioritise MVP
